src/App.js

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import './App.css';
+import Header from './components/Header';
 
 function App() {
-  return <div className="quote">Breaking Bed</div>;
+  return (
+    <div>
+      <Header />
+    </div>
```

Comment:
you can also do  ```<></> ``` instead of ```<div></div>``` so you will not have so manny unnecessary divs 🙂

Reply:
danke!